src/parser/__init__.py

```diff
@@ -7,180 +8,215 @@
 import sys
 import importlib
 from bs4 import BeautifulSoup
+from typing import Tuple, List, Optional
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+parent_dir = os.path.dirname(current_dir)
+project_root = os.path.dirname(parent_dir)
 
 class Parser:
 
     def __init__(self):
         pass
 
-    def get_class_and_method(self, county):
+    def configure_logger(self):
+        logger = logging.getLogger(name="pid: " + str(os.getpid()))
+        logging.basicConfig(
+            level=logging.INFO,
+            format='%(asctime)s - %(levelname)s - %(message)s',
+            handlers=[
+                logging.FileHandler("parser_log.txt"),
+                logging.StreamHandler() 
+            ]
+        )
+        logger.info("Logger configured")
+        return logger
+    
+    def get_class_and_method(self, logger, county: str, test = False) -> Tuple[Optional[object], Optional[callable]]:
+        if test:
```

Comment:
Again, nothing to do here, but a small note. I think in the next evolution of this code, instead of passing a test field, we'd instead pass all of the fields via unit test that the method might need. For example, if the test parser only parses a specific mock HTML stored in the test files folder, instead of putting the HTML folder path-designating logic within the code pivoting based on whether test is true, we'd instead pass the method the HTML folder path entirely. This reason being, when we test, we want to test the method logic itself rather than test only the test version. LMK if that doesn't make sense.